src/character.cpp

```diff
@@ -7264,6 +7264,25 @@ void Character::recalculate_enchantment_cache()
             }
         }
     }
+
+    // get from traits/ mutations
+    for( const auto &mutMap : my_mutations )
```

Comment:
can you remove the auto here in favor of the class? i think this is `mutation_branch`?

Reply:
actually its a element of a map `<mutation_id, mutation_data>`, i'll look into c++ syntax how to declare that, i'll keep in mind that you avoid autos.
I'm planing to work on this on Friday again

Reply:
itd be a std::pair<mutation_id, mutation_data>